images/README.md

```diff
@@ -1,18 +1,14 @@
 # Dev Images
 
-This folder contains a docker-compose file for development. To create and start the service execute `docker-compose up -d`.
-This will produce 2 following images:
+This folder contains a docker-compose file for development. It creates and starts the service which has two containers.
```

Comment:
'It creates and starts the service which has two containers.' -> 'It starts 2 containers Keycloak and PostgreSQL.